js/details.js

```diff
@@ -11,7 +11,10 @@ function show_probe( e, text ) {
 }
 
 function show_details( properties ) {
+	var pan_offset = !$( "body" ).hasClass( "details" );
 	$( "body" ).addClass( "details" );
+	resize();
+	if ( pan_offset ) map.panBy([ $( "aside" ).outerWidth()/2, 0 ],{animate:false});
```

Comment:
Line is too long.<br>Identifier 'pan_offset' is not in camel case.<br>Expected '{' and instead saw 'map'.<br>'map' is not defined.